ui/app/AppLayouts/Onboarding2/OnboardingFlow.qml

```diff
@@ -12,13 +14,17 @@ SQUtils.QObject {
 
     required property StackView stackView
 
+    required property var loginAccountsModel
+
     required property int keycardState
     required property int addKeyPairState
     required property int syncState
     required property var seedWords
     required property int remainingPinAttempts
     required property int remainingPukAttempts
 
+    required property bool isBiometricsLogin // FIXME should come from the loginAccountsModel for each profile separately?
```

Comment:
hmmm good question, I've not idea where it should come from. Need to check the code.

But I don't see it in `accounts.sql` database 🤔 
<img width="691" alt="image" src="https://github.com/user-attachments/assets/2dca9c13-33a2-48f1-9840-5d133360e184" />

Reply:
looks like content for new task, right?

Reply:
Reposting here; the old code had sth like:
```qml
isBiometricsLogin: localAccountSettings.storeToKeychainValue === Constants.keychain.storedValue.store
```

Reply:
@igor-sirotin it also seems we have an issue around this already: https://github.com/status-im/status-desktop/issues/15167